bindings/nodejs/upgrade.md

```diff
@@ -1,3 +1,33 @@
+# Unreleased
+
+## Breaking change
+
+### Public API
+
+Now, the `append` operation has been removed. You can use below code instead.
+
+```js
+op.write("path/to/file", Buffer.from("hello world"), { append: true });
+```
+
+## New features
```

Comment:
Upgrade guide only need to record breaking changes.

Reply:
Oh, my bad.